uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/ConfigurationsEditorDataFormatTest.java

```diff
@@ -250,17 +222,20 @@ public void testEditDataFormat() {
 	 */
 	@Test
 	public void testDeleteDataFormat() {
+		createDataFormat();
+		String[] path = { ROOT, dataFormatName + " (Data Format)" };
+		WaitCondition wait = new TreeHasItem(new DefaultTree(editor), path);
+		new WaitUntil(wait, TimePeriod.MEDIUM, false);
```

Comment:
Is there any reason to ignore the potential timeout exception? On the next line we throw an error anyway.